app/views/dashboard/participant_guide.html.haml

```diff
@@ -28,7 +28,9 @@
           %li
             = raw t('participant_student_guide.attendance_policy.rule5')
           %li
-            = t('participant_student_guide.attendance_policy.rule6')
+            = raw t('participant_student_guide.attendance_policy.rule6')
```

Comment:
As rule6 doesnt contain ant HTML `raw` is not needed here

Reply:
oh that's what it means, cheers. Will update :)

Reply:
Thanks @KimberleyCook !

Reply:
@despo updated.

Reply:
@KimberleyCook looks good. Can you rebase please!